src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    class="form"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text"
+          name="Surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text"
            name="Surname"
            autocomplete="off"
          >
```
let's work on code style here and everywhere else

Reply:
thanks, done